python/setup.py

```diff
@@ -1,6 +1,7 @@
 """Set up the ImJoy-Engine imjoy package."""
 import json
 import os
+import sys
```

Comment:
The standard way of requiring certain python versions is to set `python_requires` parameter in the `setup` call.

https://packaging.python.org/guides/distributing-packages-using-setuptools/#python-requires